interface/ceed-operator.c

```diff
@@ -1552,6 +1545,9 @@ int CeedOperatorApplyAdd(CeedOperator op, CeedVector in, CeedVector out, CeedReq
         CeedCall(CeedOperatorApplyAdd(sub_operators[i], in, out, request));
       }
     }
+  } else if (op->num_elem) {
+    // Standard Operator
+    CeedCall(op->ApplyAdd(op, in, out, request));
```

Comment:
Similar here. Not sure the likelihood of both `op->is_composite` and `op->num_elem` being NULL though.